django/cantusdb_project/main_app/urls.py

```diff
@@ -7,6 +7,7 @@
     PasswordResetCompleteView,
 )
 from main_app.views import views
+from main_app.views import redirect
```

Comment:
Seems strange that the others import the functions directly but these two use the module prefix...